mm2src/coins/z_coin/storage/z_params/indexeddb.rs

```diff
@@ -69,13 +69,13 @@ impl ZcashParamsWasmInner {
 pub(crate) struct ZcashParamsWasmImpl(SharedDb<ZcashParamsWasmInner>);
 
 impl ZcashParamsWasmImpl {
-    pub(crate) async fn new(ctx: &MmArc) -> MmResult<Self, ZcoinStorageError> {
-        Ok(Self(ConstructibleDb::new(ctx).into_shared()))
+    pub(crate) async fn new(ctx: &MmArc, db_id: Option<&str>) -> MmResult<Self, ZcoinStorageError> {
+        Ok(Self(ConstructibleDb::new(ctx, db_id).into_shared()))
```

Comment:
Here in new() we pass db_id as Option
but in lock_db(&self) few lines below we always call get_or_initialize(None)
is this correct?